test/framework/src/main/java/org/elasticsearch/test/rest/section/Assertion.java

```diff
@@ -66,7 +66,8 @@ protected final Object getActualValue(RestTestExecutionContext executionContext)
 
     @Override
     public final void execute(RestTestExecutionContext executionContext) throws IOException {
-        doAssert(getActualValue(executionContext), resolveExpectedValue(executionContext));
+        Object actualValue = getActualValue(executionContext);
+        doAssert(actualValue, resolveExpectedValue(executionContext));
```

Comment:
Is there a reason for this change that I'm overlooking? If not, can you revert? I suspect it's something you used while debugging so you could set a breakpoint and inspect `actualValue`? :wink: